### ISSUE TITLE
[BUG] Accessibility - Web App - Buttons should have text content - invisible button in Orientation screens missing label

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5141]
**Describe the bug**
Buttons should have text content - invisible button in Orientation screens missing label.

Buttons must have content. On the Orientation pages using Ainspector tool, I could see an invisible button, that is missing text content, label. A label associated with a form control ensures that information about the form control is spoken by screen readers when it receives focus.
[WCAG 3.3.2 Labels or Instructions](https://www.w3.org/TR/WCAG21/#labels-or-instructions)

**To Reproduce**
1. Go to app.illinois.edu in Firefox browser with AInspector installed
2. Run it on orientation screens
3. See error for this button

**Expected behavior**
Buttons should have text content.

**Actual behavior** 
Invisible button in Orientation screens missing label.

**Screenshots**

![Image](https://github.com/user-attachments/assets/08d7e60a-dd20-4654-9e5a-e2cc4a0c9b98)

**PC:**
•	OS: Windows 11
•	Device: Dell laptop
•	Browser: Chrome 135.0.7049.116/Firefox 138.0.1

**Additional context**

**Comment by acovher2** [comment]
@dobromirdobrev 
You are 100% correct, this is a garbage ticket I must have written whilest very low on caffeine. Please feel free to @ me and reject ticket's that are like this. I apologize for the duplication. I will retest the mobile scenario on iOS and open a separate issue. I've update 5149 with retest comments as well.
